screener/migrations/0122_screen_has_nc_medicare_savings.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.22 on 2025-11-06 01:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("screener", "0121_screen_has_heap"),
+    ]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)